mysql-test/mysql-test-run.pl

```diff
@@ -6228,10 +6228,18 @@ sub gdb_arguments {
      return;
   }
 
+  my $term_cmd = $ENV{MTR_TERM} || "xterm -title %title% -e";
+  my @term_args = split / /, $term_cmd;
+  my $term_exe = shift @term_args;
```

Comment:
I would also add some handling for the case when `MTR_TERM` is defined but is an empty string (not sure how `split` and `shift` will work in this case).

Reply:
Done, it reports an error correctly now.